### ISSUE TITLE
Override database_names in MongoConnection

### ISSUE THREAD

**Issue by merwok** [hadrien/pyramid_mongokit#3]
I’m using multiple databases thanks to MongoConnection, and to get all database names related to the DB prefix I need to filter the result of database_names myself:

```
[name for name in conn.database_names() if name.startswith(conn.db_prefix)]
```

MongConnection could override database_names to do this for me.

**Comment by hadrien** [comment]
It is a good idea but I would not override `database_names`. Would `prefixed_database_names` be too long?

**Comment by hadrien** [comment]
Fixed by 2dc020bc5b3f699ff6548c7847c99e62b9911482 with `MongoConnection.prefixed_database_names` method.
